src/telegram-types.ts

```diff
@@ -1,11 +1,54 @@
 /** @format */
 
-import * as TT from 'typegram'
-export * from 'typegram'
+import {
+  Chat,
+  InlineKeyboardMarkup,
+  LabeledPrice,
+  Message,
+  Typegram,
+} from 'typegram'
 
-export type ChatAction = TT.Opts<'sendChatAction'>['action']
+export * from 'typegram/callback'
+export * from 'typegram/inline'
+export * from 'typegram/manage'
+export * from 'typegram/message'
+export * from 'typegram/passport'
+export * from 'typegram/payment'
+export * from 'typegram/update'
 
-export type ChatType = TT.Chat['type']
+export interface InputFileByPath {
+  source: string
+}
+export interface InputFileByReadableStream {
+  source: NodeJS.ReadableStream
+}
+export interface InputFileByBuffer {
+  source: Buffer
+}
+export interface InputFileByURL {
+  url: string
+  filename?: string
+}
```

Comment:
Is this correct? I'm not sure if this accurately describes the `InputFile`s that `telegraf` can handle.

Reply:
@dotcypress ?